lib/system/nimscript.nim

```diff
@@ -235,20 +235,26 @@ proc cpDir*(`from`, to: string) {.raises: [OSError].} =
     copyDir `from`, to
     checkOsError()
 
-proc exec*(command: string) =
-  ## Executes an external process.
+proc exec*(command: string) {.
+  raises: [OSError], tags: [ExecIOEffect].} =
+  ## Executes an external process. If the external process terminates with
+  ## a non-zero exit code, an OSError exception is raised.
+  ## Note: If you need a version of ``exec`` that returns the exit code
+  ## and text output of the command, you can use ``gorgeEx`` from the
```

Comment:
A link here would be nice, and while we're at it, we should put the note in a separate line/paragraph:

```
  ## Executes an external process. If the external process terminates with
  ## a non-zero exit code, an OSError exception is raised.
  ##
  ## **Note:** If you need a version of ``exec`` that returns the exit code
  ## and text output of the command, you can use `system.gorgeEx
  ## <system.html#gorgeEx,string,string,string>`_.
```